server/config-loader.ts

```diff
@@ -43,6 +69,40 @@ export function load(invalidate = false) {
 		}
 	}
 
+	if ('subprocesses' in config) {
+		// Leniently accept all falsy values, including `null`.
+		if (!config.subprocesses || config.subprocesses === 1) {
+			config.subprocesses = Object.fromEntries(
+				processTypes.map(k => [k, ~~Boolean(config.subprocesses)])
+			) as Record<ProcessType, number>;
```

Comment:
Man I just really don't like `~~` and `~~Boolean` is just weird (and also we use `!!` rather than `Boolean`).


```suggestion
		// Leniently accept all falsy values, including `null`.
		config.subprocesses ||= 0;
		if (config.subprocesses === 0 || config.subprocesses === 1) {
			config.subprocesses = Object.fromEntries(
				processTypes.map(k => [k, config.subprocesses])
			) as Record<ProcessType, number>;
```